TADashboard/src/main/java/page_objects/DataProfilesGeneralSettingsPage.java

```diff
@@ -1,12 +1,15 @@
 package page_objects;
 
 import com.logigear.control.common.imp.Button;
-import com.logigear.control.common.imp.Link;
-import com.logigear.driver.DriverUtils;
+import com.logigear.control.common.imp.ComboBox;
+import com.logigear.control.common.imp.TextBox;
 
 public class DataProfilesGeneralSettingsPage {
     private Button btnNext = new Button("css=input[value='Next']");
     private Button btnFinish = new Button("css=input[value='Finish']");
+    private TextBox txtName = new TextBox("css=#txtProfileName");
```

Comment:
how about ID?

Reply:
Yeah, of cource. I will fix it for next commit.